cypress/integration/projectSettings/project_settings.ts

```diff
@@ -41,15 +41,44 @@ describe("Repo Settings", () => {
       cy.dataCy("navitem-github-commitqueue").click();
     });
 
-    it("Successfully saves a patch definition", () => {
+    it("Updates a patch definition", () => {
       cy.dataCy("add-button").contains("Add Patch Definition").parent().click();
 
       cy.dataCy("variant-tags-field").find("button").click();
       cy.dataCy("variant-tags-input").first().type("vtag");
 
       cy.dataCy("task-tags-field").find("button").click();
       cy.dataCy("task-tags-input").first().type("ttag");
+    });
+
+    it.only("Toggling disable commit queue hides inputs", () => {
```

Comment:
Is this `it.only` intentional? (i.e. won't be running any tests in this file except this one?)

Reply:
Ah good catch 😰 thank you!!